app/Model/MainModel.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Model;
+
+use Illuminate\Support\Facades\DB;
+use Illuminate\Database\Eloquent\Model;
+
+class MainModel extends Model
+{
+    protected $name = 'name';
+    /**
+     * Find Model by String
+     *
+     * @param string $string input string to find
+     *
+     * @return App\Model
+    */
+    public function findByString($string)
+    {
+        $obj = DB::table($this->table)->where($this->name, $string)->get();
```

Comment:
sửa chỗ này lại thành ở dưới này nè để cái đó nó chứa cái name là đc rồi 
$obj = DB::table($this->table)->where($this->name,  'like', '%'.$string.'%')->get();